RTClientSDK.Net.Example/Program.cs

```diff
@@ -18,12 +18,14 @@ static void Main(string[] args)
             //Example example = new ExampleSkeleton(mRtProtocol, mIpAddress);
             //Example example = new Example3D(mRtProtocol, mIpAddress);
             //Example example = new ExampleImage(mRtProtocol, mIpAddress);
-            Example example = new Example2D(mRtProtocol, mIpAddress);
+            //Example example = new Example2D(mRtProtocol, mIpAddress);
             //Example example = new Example6D(mRtProtocol, mIpAddress);
             //Example example = new ExampleGaze(mRtProtocol, mIpAddress);
             //Example example = new ExampleEyeTracker(mRtProtocol, mIpAddress);
             //Example example = new ExampleTimecode(mRtProtocol, mIpAddress);
             //Example example = new ExampleUDP(mRtProtocol, mIpAddress);
+            //Example example = new ExampleUDP(mRtProtocol, mIpAddress);
+            Example example = new ExampleSetGeneralSettings(mRtProtocol, mIpAddress);
```

Comment:
Should this be uncommented out?

Reply:
I thought it made sense to have the most recent example enabled.